src/app/shared/navbar/navbar.ts

```diff
@@ -17,8 +19,21 @@ const SECTIONS_KEYS = Object.keys(SECTIONS);
   templateUrl: './navbar.html',
   styleUrls: ['./navbar.scss']
 })
-export class NavBar {
+export class NavBar implements OnDestroy {
+  private subscriptions = new Subscription();
```

Comment:
A common approach for unsubscribing is to have an onDestroy observable, i.e.
```
private readonly onDestroy = new Subject<void>();

...
router.events.pipe(takeUntil(this.onDestroy), ...).subscribe(...);
...

ngOnDestroy() {
  this.onDestroy.next();
  this.onDestroy.complete();
}
```

Up to you - this is more or less an FYI. It's more useful when you have multiple subscriptions that you need to unsubscribe from or as a project-wide convention.

Reply:
I misread how this was being used. I like this `subscriptions.add` too :)